src/BizHawk.Emulation.Cores/Consoles/Sony/PSP/PPSSPP.ISettable.cs

```diff
@@ -0,0 +1,65 @@
+using BizHawk.Common;
+using BizHawk.Emulation.Common;
+
+namespace BizHawk.Emulation.Cores.Consoles.Sony.PSP
+{
+	public partial class PPSSPP : ISettable<PPSSPP.Settings, PPSSPP.SyncSettings>

```

Comment:
If there are no settings, use `ISettable<object, ...>`, and likewise for sync settings, and if there are neither then I think you can just omit the interface.

Reply:
There are no settings now, but there may be in the future. For this reason, I think it's better to preserve the structure to allow these, even if currently empty.

Reply:
Preserve the copy-pasted boilerplate? When you don't plan on touching the core for a while?

Reply:
removed